cfpb/cfgov-refresh
# Stop storing ZIP codes as integers

Our EventPage model stored ZIP codes as an IntegerField, which removed
significant leading zeros. This creates a string version and restores
missing zeros.

The migration doesn't need a user because the new field doesn't generate
a page revision, so the page doesn't need to be published for the fix
to show up.

## Testing
* Pull in recent prod data
* Check this page locally to see a bad ZIP code: <http://localhost:8000/about-us/events/archive-past-events/field-hearing-on-arbitration/>
* Run the migration in this branch with `./cfgov/manage.py migrate`
* Recheck the page above. It should have a 5-digit ZIP code.
